src/components/Reactions/ReportActionItemEmojiReactions.js

```diff
@@ -108,6 +113,7 @@ function ReportActionItemEmojiReactions(props) {
             reactionCount,
             hasUserReacted,
             onReactionListOpen,
+            hasPendingAction: reaction.pendingAction,
```

Comment:
`hasPendingAction` is a confusing name, it makes me thing this is gonna be a bool

Reply:
I agree we can use plain `pendingAction`